packages/app/src/app/pages/Dashboard/Content/routes/Liked/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { SelectionProvider } from 'app/pages/Dashboard/Components/Selection';
+import { useAppState, useActions } from 'app/overmind';
+import { sandboxesTypes } from 'app/overmind/namespaces/dashboard/types';
+import { Header } from 'app/pages/Dashboard/Components/Header';
+import { VariableGrid } from 'app/pages/Dashboard/Components/VariableGrid';
+import { DashboardGridItem, PageTypes } from 'app/pages/Dashboard/types';
+import { getPossibleTemplates } from '../../utils';
+
+export const Liked = () => {
+  const {
+    activeTeam,
+    dashboard: { sandboxes, getFilteredSandboxes },
+  } = useAppState();
+  const {
+    dashboard: { getPage },
+  } = useActions();
+
+  React.useEffect(() => {
+    getPage(sandboxesTypes.LIKED);
+  }, [activeTeam, getPage]);
+
+  const items: DashboardGridItem[] = sandboxes.LIKED
+    ? getFilteredSandboxes(sandboxes.LIKED).map(sandbox => ({
+        type: 'sandbox',
+        sandbox,
+      }))
+    : [{ type: 'skeleton-row' }, { type: 'skeleton-row' }];
+
+  const pageType: PageTypes = 'liked';
+  return (
+    <SelectionProvider page={pageType} activeTeamId={activeTeam} items={items}>
+      <Helmet>
+        <title>Liked Sandboxes - CodeSandbox</title>
+      </Helmet>
+      <Header
+        title="Liked Sandboxes"
+        activeTeam={activeTeam}
+        templates={getPossibleTemplates(sandboxes.LIKED)}
+        showViewOptions
+        showFilters
+      />
+
+      <VariableGrid items={items} page={pageType} />
```

Comment:
Do we define somewhere that these sandboxes are "read-only" eg. that we can't drag them or do actions on them? I haven't tested yet, not sure if we define it in code somewhere.

Reply:
We check in the menu if the author is equal to the login user so it all happens automatically

Reply:
❤️ 🎉  Super nice!